packages/core/src/components/meter/meter.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, Element, h, Host, Prop } from '@stencil/core'
+
+@Component({
+  tag: 'atom-meter',
+  styleUrl: 'meter.scss',
+  shadow: true,
+})
+export class AtomMeter {
+  @Prop() type?: 'success' | 'neutral' | 'warning' | 'danger' = 'neutral'
+  @Prop() size?: 'small' | 'large' = 'large'
+  @Prop() title: string
+  @Prop() centerTitle?: boolean = false
+  @Prop() min: number = 0
```

Comment:
Is min also optional, isn't it?

Reply:
I think so, I don'ts realy it makes sense to set a default value since max is also optional.

Made: https://github.com/juntossomosmais/atomium/pull/695/commits/be97b7d960a8e4093f285b72c11083000b73eab0

Reply:
sorry, @gabrielduete . I wasn't clear. I agree with you, but it was missing the `?`.

  @Prop() min`?`: number = 0